ARK Server Manager/OpenRCON.xaml.cs

```diff
@@ -48,8 +52,32 @@ public OpenRCON()
             this.DataContext = this;
         }
 
+        public static int GetMaxPlayers()
+        {
+            
+            JsonSerializer serializer = new JsonSerializer();
+            using (var client = new WebClient())
```

Comment:
Ensure this property does not run on the UI thread as it performs network calls.